kong/tracing/instrumentation.lua

```diff
@@ -0,0 +1,229 @@
+local table       = table
+local pack        = table.pack
+local unpack      = table.unpack
+local fmt         = string.format
+local base        = require "resty.core.base"
+local to_hex      = require "resty.string".to_hex
+local pdk_tracer  = require "kong.pdk.tracing".new()
+-- TODO(mayo): Bring the propagation library into PDK and make it possible to register custom handlers
+local propagation = require "kong.plugins.zipkin.tracing_headers"
```

Comment:
i would suggest to move the `tracing_headers.lua` file into `kong/tracing/`, and let zipkin plugin import this file, instead of doing the opposite.

Reply:
Agree, but the `tracing_headers.lua` is not fully compatible with the new API, `proxy_span:each_baggage_item` is NYI, I'd like to refactor the Zipkin plugin to use the same propagation API later.